server/server/src/billing/stripe_service.rs

```diff
@@ -0,0 +1,196 @@
+use crate::billing::billing_model::StripeUserInfo;
+use crate::billing::stripe_client;
+use crate::{keys, StripeWebhookError};
+use crate::{ClientError, ServerError, ServerState};
+use deadpool_redis::Connection;
+use google_androidpublisher3::hyper::body::Bytes;
+use google_androidpublisher3::hyper::header::HeaderValue;
+use libsecp256k1::PublicKey;
+use lockbook_models::api::{PaymentMethod, StripeAccountTier, UpgradeAccountStripeError};
+use log::info;
+use redis_utils::converters::{JsonGet, JsonSet};
+use std::ops::Deref;
+use std::str::FromStr;
```

Comment:
This might have understandably gotten lost in conflict resolution, but here in this file, just re-adopt these changes: https://github.com/lockbook/lockbook/pull/1149/files#diff-8e3051f69d5666b86afd3e9818eeec3c8aec959c9a13bb8847656d09690679aa